packages/quantic/force-app/main/default/lwc/quanticSearchBox/templates/expandableSearchBox.css

```diff
@@ -18,6 +18,7 @@ textarea.searchbox__input {
   border: none;
   box-shadow: none;
   overflow-x: clip;
+  margin-right: 0.8rem;
```

Comment:
Looks fine to me, but is the 0.8rem an arbitrary value or is there a reason behind it?

Reply:
It's the same value as the lightning design system value that is on the sides of the icon next to it 🤷‍♂️